### ISSUE TITLE
Zooming/base pairing probability/partition support

### ISSUE THREAD

**Issue by herber4** [fjossinet/RNArtist#15]
Hello RNArtist, 

Love the visualization and looking forward to following this package as it grows. 

Can you slow down the zoom in/out feature? if im not careful I can completely lose the molecule bc of how fast it zooms, haha. 

Another question, are you planning on integrating the ability to visualize base pairing probability files? 

Thanks, 

Austin Herbert

**Comment by herber4** [comment]
oh, and also a click and drag option to reposition the molecule on screen would be super cool!

**Comment by fjossinet** [comment]
Hi. Did you install the last version from this page: https://www.jdeploy.com/~rnartist

Could you give it a try because many things have been modified and fixed. Thanks

**Comment by herber4** [comment]
Hello! 

I have downloaded and am unable to load structures in databases with dot bracket and CT files generated from RNAstructure command line program.

here's the code I ran:
Fold ANKHD1_chr5_140438044_140438944.fasta mfe.ct -md 
[mfe.ct.txt](https://github.com/fjossinet/RNArtist/files/13054104/mfe.ct.txt)
250 -mfe -sh ANKHD1_chr5_140438044_140438944_ANKHD1_chr5_140438044_140438944.shape

and the ct file I generated is attached (added .txt extension for GitHub to take)

looking forward to hearing back from you! 

thanks

**Comment by herber4** [comment]
Hello, 

I have been using the version from this installer: https://www.jdeploy.com/~rnartist

I am on a Mac desktop, intel chip, Ventura v13.5.2 and I installed the MAcOS 10.14+

**Comment by fjossinet** [comment]
At now, everything is working fine for Vienna files only. But the nest release (very few days) will do the stuff for bpseq and ct. Meanwhile, I can send u a Vienna version of your data. Or the svg. Let me know.

**Comment by fjossinet** [comment]
Hi. I have released the new version (1.1.0). Give it a try and let me know. I was able to load your file mfe.ct in the bracket notation panel.

**Comment by herber4** [comment]
looks great, looking forward to using this more often! thanks for the help :)

**Comment by herber4** [comment]
hi, 

I still think having a mode where we can click around the art, move the structure around with mouse. like when zoomed in on a structure being able to reposition the structure within that zoom by clicking and dragging with the mouse would be useful. 

thanks!

**Comment by fjossinet** [comment]
You can translate the view with your mouse by keeping the right mouse button pressed and dragged. Concerning your request, I was thinking about the ability to define an area with the left button pressed and dragged to zoom in.

**Comment by herber4** [comment]
yeah, that sounds like a great idea. We are working with experimental derived structures for introns, so it's nice to be able to rapidly move around the large structures. Also, being able to generate circular plots from a regular plot would be a cool function.
